pkg/kafsar/kafka_impl.go

```diff
@@ -129,3 +246,25 @@ func (k *KafkaImpl) SaslAuthConsumerGroup(req service.SaslReq, consumerGroup str
 func (k *KafkaImpl) Disconnect(addr net.Addr) {
 	panic("implement me")
 }
+
+func (k *KafkaImpl) createConsumer(topic, groupId string) (chan pulsar.ConsumerMessage, pulsar.Consumer, error) {
+	channel := make(chan pulsar.ConsumerMessage, k.kafsarConfig.ConsumerReceiveQueueSize)
+	options := pulsar.ConsumerOptions{
+		Topic:                       topic,
+		SubscriptionName:            groupId,
```

Comment:
add callback convert method to allow user config

Reply:
although add the convert method,  but here doesn't use